packages/app/src/pages/_app.tsx

```diff
@@ -34,7 +34,9 @@ type AppPropsWithLayout = AppProps & {
 
 const App = ({ Component, pageProps }: AppPropsWithLayout) => {
   const getLayout = Component.getLayout || (page => page);
-
+  useEffect(() => {
+    printBuildInfo();
+  }, []);
```

Comment:
I think the better practice is put printBuildInfo into the head

like 

```html
<head>
  <meta name="affine:version" content="xxx">
</head>
```

```js
var meta = document.createElement('meta');
meta.name = "affine:version";
meta.content = publicRuntimeConfig.EDITOR_VERSION;
document.getElementsByTagName('head')[0].appendChild(meta);
```

Reply:
playwright how to get this value?

Reply:
also to remove the useEffect, because this log is outside of the react lifecircle

```js
// top of the entry point file
if (typeof window !== 'undefined') {
  document.addEventListerner('load', () => {
    // xxx
  })
}
```

Reply:
> also to remove the useEffect, because this log is outside of the react lifecircle
> 
> ```js
> // top of the entry point file
> if (typeof window !== 'undefined') {
>   document.addEventListerner('load', () => {
>     // xxx
>   })
> }
> ```

@lawvs  what's your suggestion?